cg/meta/workflow/microsalt.py

```diff
@@ -258,16 +258,17 @@ def get_deliverables_to_store(self) -> list:
         deliverables_to_store = []
         for case_object in self.db.cases_to_store(pipeline="microbial"):
             deliverables_file = self.get_deliverables_file_path(order_id=case_object.name)
-            if Path(deliverables_file).exists():
-                deliverables_to_store.append(deliverables_file)
+            if not Path(deliverables_file).exists():
+                continue
+            deliverables_to_store.append(deliverables_file)
```

Comment:
Do you need to convert back to str here? I'm not sure what you do with them in the next step...
Whichever you choose you can update the return type to be more specific like

```python
def get_deliverables_to_store(self) -> List[str]:
or
def get_deliverables_to_store(self) -> List[Path]:
```
Don't forget to import `from typing import List`

Reply:
fixed